intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -216,6 +216,17 @@ \section{First Time connection to the \hpc}
 Are you sure you want to continue connecting (yes/no)? %\strong{\emph{yes}}%
 \end{prompt}
 
+A possible error message is:
+\begin{prompt}
+%\shellcmd{Permission denied (publickey,gssapi-keyex,gssapi-with-mic).}%
+\end{prompt}
+
+If you previously saved your private key in the non default location (.ssh/id_rsa) you can get the error message
+In this case use:
+\begin{prompt}
+%\shellcmd{ssh -i <path/to/location/of/private/key \userid{}@\loginnode{}}%
```

Comment:
remove the `<` (there's no closing `>`, and this would confuse people anyway, the `/path/to/...` should make it clear it should be replaced